src/main/java/org/web3j/codegen/SolidityFunctionWrapper.java

```diff
@@ -73,38 +73,32 @@ public void generateJavaFiles(String contractName, String bin, String abi,
             throws IOException, ClassNotFoundException {
         String className = Strings.capitaliseFirstLetter(contractName);
 
-        FieldSpec fieldSpec = FieldSpec.builder(String.class, "BINARY")
-                .addModifiers(Modifier.PRIVATE, Modifier.FINAL, Modifier.STATIC)
-                .initializer("$S", bin)
+        TypeSpec.Builder classBuilder = createClassBuilder(className, bin);
+
+        classBuilder.addMethod(buildConstructor(Credentials.class, "credentials"));
```

Comment:
Sorry - I missed this. These strings should be defined as constants as per in the original SolidityFunctionWrapperGenerator.